bigquery_etl/shredder/bug1751979.py

```diff
@@ -249,3 +249,119 @@
     )
 ) AS payload)
 """
+
+BUG_1751979_MAIN_SUMMARY_V4_UDFS = r"""
+CREATE TEMP FUNCTION sanitize_search_counts_ms(
+  input ARRAY<STRUCT<engine STRING, source STRING, count INT64>>
```

Comment:
I am realizing now that this is potentially opening us up to duplicate keys, which could cause pain for future analyses. We could have two entries ("google", "in-content:foo", 1) and ("google", "in-content:bar", 1) which should both end up with the code at the end being scrubbed to `other.scrubbed`. We'll have this same issue in other derived tables, so it may be worth turning this into an aggregation as part of this function.

Reply:
added suggestions that accomplish this